src/Config.cpp

```diff
@@ -64,10 +64,8 @@ bool	Config::saveVariable(const std::string& line)
 
 	if (line[i++] != ';' || i < line.length()) // [;]
 		return (false);
-	// TODO: line[i + 1] != '\0'
 
-	_tmpServer.addVariable(key, value);
-	return (true);
+	return (_tmpServer.addVariable(key, value, false));
```

Comment:
Can we have _tmpServer as _tmpBlockServer?

Reply:
Sir, yes, Sir!